src/service/ConfigService.ts

```diff
@@ -219,10 +221,38 @@ export class ConfigService {
                 presetData.nemesisSeedFolder,
                 `Is the provided preset nemesisSeedFolder: ${presetData.nemesisSeedFolder} a valid seed folder?`,
             );
+            logger.info(`Using custom nemesis seed folder in ${presetData.nemesisSeedFolder}`);
             await BootstrapUtils.generateConfiguration({}, presetData.nemesisSeedFolder, nemesisSeedFolder);
             return;
         }
 
+        if (presetData.nemesisSeedUrl) {
+            let downloading = true;
+            process.on('SIGINT', () => {
+                if (downloading) {
+                    logger.error('The seed cannot be downloaded. Processed has been terminated...');
+                    process.exit();
+                }
+            });
+            while (true) {
+                try {
+                    const zipFile = `${this.params.preset}-nemesis.zip`;
+                    await BootstrapUtils.download(presetData.nemesisSeedUrl, zipFile);
+                    await ZipUtils.unzip(zipFile, 'seed', nemesisSeedFolder);
+                    await this.validateSeedFolder(
+                        nemesisSeedFolder,
+                        `Is the nemesisSeedUrl: ${presetData.nemesisSeedUrl} a valid seed zip file?`,
+                    );
+                    downloading = false;
+                    return;
+                } catch (e) {
+                    logger.warn(`Seed ${presetData.nemesisSeedUrl} cannot be downloaded. Error ${e.message}`);
+                    logger.warn(`The ${presetData.preset} network hasn't been launched yet!!!. Retrying in 10 seconds ....`);
+                    await BootstrapUtils.sleep(10000);
```

Comment:
Isn't 10s too frequent? We might have 600 nodes pinging that URL every 10s during a week...